SilasKenneth/fast-food-fast-ui
# [Finishes #160131466] Create a signup page

#### What this pull request does
Creates the page where users are able to login
##### Description of the task to be completed
 Create a page where a user can create an account
    -  Add the form for the signup
    - Add links for the user to choose other options
##### Relevant pivotal tracker story
[#160131466](https://www.pivotaltracker.com/n/projects/2194200/stories/160131466)
##### How to test
  - Clone the repo ``git clone https://github.com/SilasKenneth/fast-food-fast-ui.git``
  -  Go to the fast-food-fast-ui directory and head to UI folder, ```git checkout the ft-create-account-160131466```
  - Open the additem.html file in your browser and that should work

#### Screenshot
![Create account](https://user-images.githubusercontent.com/11555354/44915521-d3081c80-ad3b-11e8-8202-c2b90d75110e.png)
